py/server/deephaven/stream/kafka/__init__.py

```diff
@@ -3,3 +3,33 @@
 #
 
 """ The kafka module provides the utilities to both consume and produce Kakfa streams. """
+
+from typing import Dict, List
+
+import jpy
+
+from deephaven import DHError
+from deephaven.jcompat import j_list_to_list, j_properties
+
+_JKafkaTools = jpy.get_type("io.deephaven.kafka.KafkaTools")
+
+def topics(kafka_config: Dict) -> List[str]:
+    """Returns a list of topic names available from Kafka.
+
+    Args:
+        kafka_config (Dict): configuration for the associated Kafka consumer.
```

Comment:
Not sure where the 'associated Kafka consumer' comes from.  Should it be a configuration for the targeted Kafka cluster?

Reply:
This is exactly the same that we say in the documentation for produce and consume:
```
def produce(
    table: Table,
    kafka_config: Dict,
    topic: str,
    key_spec: KeyValueSpec,
    value_spec: KeyValueSpec,
    last_by_key_columns: bool = False,
) -> Callable[[], None]:
    """Produce to Kafka from a Deephaven table.                                                                                                 
                                                                                                                                                
    Args:                                                                                                                                       
        table (Table): the source table to publish to Kafka                                                                                     
        kafka_config (Dict): configuration for the associated kafka producer     
[...]
```

and


```
def consume(
        kafka_config: Dict,
        topic: str,
        partitions: List[int] = None,
        offsets: Dict[int, int] = None,
        key_spec: KeyValueSpec = None,
        value_spec: KeyValueSpec = None,
        table_type: TableType = TableType.stream(),
) -> Table:
    """Consume from Kafka to a Deephaven table.                                                                                                 
                                                                                                                                                
    Args:                                                                                                                                       
        kafka_config (Dict): configuration for the associated Kafka consumer and also the resulting table.                                      
            Once the table-specific properties are stripped, the remaining one is used to call the constructor of                               
            org.apache.kafka.clients.consumer.KafkaConsumer; pass any KafkaConsumer specific desired configuration here
[...]
```